pytentiostat/config.json

```diff
@@ -0,0 +1,26 @@
+{
+    "Experiment":{
+      "Type":"LSV"
+    },
+    "Plot":{
+      "Type":"plot"
+    },
+    "Write":{
```

Comment:
What are the options here?  Is it just turning writing on or off?  In that case it is better as ``"write": True`` so the schema is that ``write`` is  a boolean.

The code will then be 
```
if write: o.write(something)
`` 
which is much more compact and understandable